python/common/org/Python.java

```diff
@@ -501,10 +501,22 @@ public static void initializeModule(java.lang.Class cls, java.util.Map<java.lang
         __doc__ = "complex(real[, imag]) -> complex number" +
             "\n" +
             "Create a complex number from a real part and an optional imaginary part.\n" +
-            "This is equivalent to (real + imag*1j) where imag defaults to 0.\n"
-    )
-    public static org.python.types.Complex complex() {
-        throw new org.python.exceptions.NotImplementedError("Builtin function 'complex' not implemented");
+            "This is equivalent to (real + imag*1j) where imag defaults to 0.\n",
+        default_args = {"real", "imag"}
+    )
+    public static org.python.types.Complex complex(org.python.Object real, org.python.Object imag) {
+        org.python.types.Float real_val;
+        org.python.types.Float imag_val;
+        if(real instanceof org.python.types.Str && imag != null) {
```

Comment:
A completely cosmetic code style thing, but since you're going to need another commit to do a final CI build - there should be a space between `if` and the opening bracket.